corehq/apps/app_manager/models.py

```diff
@@ -5143,7 +5143,7 @@ def make_build(self, comment=None, user_id=None, previous_version=None):
         if not copy._id:
             # I expect this always to be the case
             # but check explicitly so as not to change the _id if it exists
-            copy._id = copy.get_db().server.next_uuid()
+            copy._id = uuid.uuid4().hex
```

Comment:
Ah, right. Cool. Saw this right below it in the docs you linked, wonder if it'll have an impact—at this scale it's not clear to me it wouldn't

> Impact of UUID choices: the choice of UUID has a significant impact on the layout of the B-tree, prior to compaction.
>
> For example, using a sequential UUID algorithm while uploading a large batch of documents will avoid the need to rewrite many intermediate B-tree nodes. A random UUID algorithm may require rewriting intermediate nodes on a regular basis, resulting in significantly decreased throughput and wasted disk space space due to the append-only B-tree design.
> 
> It is generally recommended to set your own UUIDs, or use the sequential algorithm unless you have a specific need and take into account the likely need for compaction to re-balance the B-tree and reclaim wasted space.